### DISCUSSION TITLE
Should we invite more people to work on this?

### DISCUSSION THREAD

**Discussion by Explosion-Scratch** [ScratchAddons/ScratchAddons#127] [Discussion]
@apple502j @Hans5958 and @Ani-Xan ?

**Comment by WorldLanguages** [comment]
Apple is already working on some nice prototype pollution "hacks" that would hopefully let us more easily make addons that enhance the Scratch editor! :)

**Comment by Explosion-Scratch** [comment]
Cool! What about the others?

**Comment by Hans5958** [comment]
I'm trying to do #71 by myself. Seems pretty easy to me.

**Comment by tb148** [comment]
Note: Apple is invited.
